### PR TITLE
sunos/*: add Dutch translation 

### PR DESCRIPTION
**Pull request by SimonVerhoeven**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.nl/sunos/devfsadm.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# devfsadm
+
+> Administratie commando voor `/dev`. Beheert de `/dev` namespace.
+> meer informatie: <https://www.unix.com/man-page/sunos/1m/devfsadm>.
```
Comment (navarroaxel) [inline pages.nl/sunos/devfsadm.md:4]:
```suggestion
> Meer informatie: <https://www.unix.com/man-page/sunos/1m/devfsadm>.
```

File: pages.nl/sunos/prctl.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# prctl
+
+> Lees of configureer de Get or set the resource controls of running processes, tasks, and projects.
+> More information: <https://www.unix.com/man-page/sunos/1/prctl>.
```
Comment (navarroaxel) [inline pages.nl/sunos/prctl.md:4]:
```suggestion
> Meer informatie: <https://www.unix.com/man-page/sunos/1/prctl>.
```

File: pages.nl/sunos/snoop.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+`snoop -d {{e1000g0}}`
+
+- Slaag de paketten op in een bestand, in plaats van ze weer te geven:
+
+`snoop -o {{filename}}`
+
+- Toon de verboze protocal layer samenvatting van de paketten in een bestand:
+
+`snoop -V -i {{filename}}`
```
Comment (navarroaxel) [inline pages.nl/sunos/snoop.md:17]: Please translate the placeholders, like `filename`, `hostname`, `port`, etc., for get this easier for non-English speakers.

**Review by navarroaxel — APPROVED** [review]

File: pages.nl/sunos/snoop.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# snoop
+
+> Network packet sniffer.
+> SunOS equivalent van tcpdump.
+> Meer informatie: <https://www.unix.com/man-page/sunos/1m/snoop>.
+
+- Bekijk de paketten van een specifieke netwerk interface:
```
Comment (kbdharun) [inline pages.nl/sunos/snoop.md:7]:
```suggestion
- Capteren de paketten van een specifieke netwerk interface:
```
Is this better? What do you think

> Note: I am not a native speaker. I checked this with Google translate.

 Is it `paketten` or `pakketten`?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have a small suggestion for this page.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**